docker/build/installers/install_python_modules.sh

```diff
@@ -29,8 +29,7 @@ apt update -y && apt install -y \
     python3-pip \
```

Comment:
python3-pip has been installed and upgraded in the `install_minimal_environment.sh` section.  Please remove it here. Besides, better install pthon3-psutil and python3-scipy stuff via pip3_install, or else there will be both system provided python modules (psutil, scipy, and their dependencies) and pip3 installed versions specified in `py3_requirements.txt`.

Reply:
done